packages/dds/tree/src/test/simple-tree/toMapTree.spec.ts

```diff
@@ -540,19 +541,36 @@ describe("toMapTree", () => {
 
 		it("Throws on schema-incompatible entries", () => {
 			const schemaFactory = new SchemaFactory("test");
-			const schema = schemaFactory.map("map", schemaFactory.string);
+			const schema = schemaFactory.object("testObject", {
+				field1: schemaFactory.string,
```

Comment:
What is the reason for this change?

Reply:
Whoops, was checking something and forgot to change it back. Reverted the change now :)